airflow/www/static/js/tree/api/useTreeData.js

```diff
@@ -29,24 +29,30 @@ import useErrorToast from '../useErrorToast';
 
 // dagId comes from dag.html
 const dagId = getMetaValue('dag_id');
-const treeDataUrl = getMetaValue('tree_data_url');
+const treeDataUrl = getMetaValue('tree_data_url') || '';
```

Comment:
This is so treeDataUrl is a string rather than undefined?

Reply:
Yes, this was to fix a test error. Let me think if there is a more appropriate way to handle that.

Reply:
Actually, this is a simple enough of a solution. We will change how we get the urls soon with [#23422](https://github.com/apache/airflow/issues/23422)